File: tests/chold/cursor.js
/* Test format */

// RESULT OF JOIN (node "object", preset default)
const obj = { one: 'one', two: [ 1, 2 ], three: { one: 'one', two: [ 1, 2 ] } };

// RESULT OF SPLIT (node "object", preset default)
const obj = {
  one: 'one',
  two: [ 1, 2 ],
  three: { one: 'one', two: [ 1, 2 ] },
};

// RESULT OF JOIN (node "object")
const arr = [ [ 1, 2, 3 ], [ 3, 2, 1 ], [ 1, 2, 3 ], [ 3, 2, 1 ] ];

// RESULT OF SPLIT (node "object")
const arr = [
  [
    1,
    2,
    3,
  ],
  [
    3,
    2,
    1,
  ],
  [
    1,
    2,
    3,
  ],
  [
    3,
    2,
    1,
  ],
];

File: tests/chold/cursor.lua
-- RESULT OF JOIN (node "block" in function_declaration), preset default
function foo () local test = { one = 'one', two = 'two' } return test end

-- RESULT OF SPLIT (node "block" in function_declaration), preset default
function foo ()
  local test = { one = 'one', two = 'two' }
  return test
end

-- RESULT OF JOIN (node "block" in if_statement), preset default
if true then return false end

-- RESULT OF SPLIT (node "block" in if_statement), preset default
if true then
  return false
end
